cPP_DSC.adoc

```diff
@@ -777,7 +782,7 @@ The following table provides the allowed choices for completion of the selection
 |{empty}[*selection*: [.underline]#ANSI X9.63, IEEE 1363a, ISO/IEC 18033-2 Part 2, SECG SEC1 sec 5.1]#
 
 |KDF
-|{empty}[*selection*: [.underline]#KDF-CTR, KDF-FB, KDF-DPI]# with concatenated keys as input using {empty}[*selection*: [.underline]#AES-128-CMAC; AES-192-CMAC; AES-256-CMAC, HMAC-SHA-1; HMAC-SHA-256; HMAC-SHA-512]# as the PRF.
+|{empty}[*selection*: [.underline]#KDF-CTR, KDF-FB, KDF-DPI]# with concatenated keys as input using {empty}[*selection*: [.underline]#AES-128-CMAC; AES-192-CMAC; AES-256-CMAC, HMAC-SHA-1; HMAC-SHA-224; HMAC-SHA-256; HMAC-SHA-384; HMAC-SHA-512; HMAC-SHA3-224; HMAC-SHA3-256; HMAC-SHA3-384; HMAC-SHA3-512]# as the PRF.
```

Comment:
Considering that HMAC-SHA-224 and HMAC-SHA-384 are listed in FCS_COP.1/KeyedHash, is there any reason not to allow them here? Also, SHA-3 is allowed by FCS_COP.1/Hash, so is there any reason not to allow those?